sources/packages/web/src/views/aest/Reports.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <full-page-container>
+    <template #header>
+      <header-navigator title="Ministry" subTitle="Reports" />
+    </template>
+    <body-header title="Export financial reports" />
+    <formio
+      formName="exportfinancialreports"
+      @loaded="formLoaded"
+      @submitted="exportReport"
+    ></formio>
+    <v-row class="justify-center m-4"
+      ><v-btn color="primary" @click="submitForm">Export CSV file</v-btn></v-row
+    >
+  </full-page-container>
+</template>
+
+<script lang="ts">
+import { ReportService } from "@/services/ReportService";
+import { ReportsFilterAPIInDTO } from "@/services/http/dto";
+import { useToastMessage } from "@/composables";
+export default {
+  setup() {
+    const toast = useToastMessage();
+    let formData: any = undefined;
```

Comment:
This is the form itself not the data of the form, right?
Please use the type `FormIOForm` instead of `any` and please remove the _data_ from the name.

Reply:
When I use the type FormIOForm, I get compilation error mentioning formData.submit() not present. 

![image](https://user-images.githubusercontent.com/54600590/170296356-8f4ec967-fd7c-4ef1-b7c9-d941f565d385.png)
Can we add submit() to the type signature?
![image](https://user-images.githubusercontent.com/54600590/170297449-16d7266a-9e9a-4116-b0d2-f87c3f292555.png)